### PR TITLE
This looks like += and -=

### PR DESCRIPTION
**Pull request by mccoyst**
Gooble gobble

### CODE REVIEW

File: plugins/counter/counter.go [hunk 1]
```diff
@@ -267,6 +267,49 @@ func (p *CounterPlugin) Message(message msg.Message) bool {
...
+			return true
+		} else if parts[1] == "-=" {
+			// -= those fuckers
+			item, err := GetItem(p.DB, subject, itemName)
+			if err != nil {
+				log.Printf("Error finding item %s.%s: %s.", subject, itemName, err)
+				// Item ain't there, I guess
+				return false
+			}
+			n := strconv.Atoi(parts[2])
```
Comment (chrissexton) [inline plugins/counter/counter.go]: Gotta handle Atoi's err value

File: plugins/counter/counter.go [hunk 2]
```diff
@@ -267,6 +267,49 @@ func (p *CounterPlugin) Message(message msg.Message) bool {
...
+
+		if parts[1] == "+=" {
+			// += those fuckers
+			item, err := GetItem(p.DB, subject, itemName)
+			if err != nil {
+				log.Printf("Error finding item %s.%s: %s.", subject, itemName, err)
+				// Item ain't there, I guess
+				return false
+			}
+			n := strconv.Atoi(parts[2])
```
Comment (chrissexton) [inline plugins/counter/counter.go]: Gotta import strconv (and handle err value)

File: plugins/counter/counter.go [hunk 3]
```diff
@@ -267,6 +267,49 @@ func (p *CounterPlugin) Message(message msg.Message) bool {
...
 			return true
 		}
+	} else if len(parts) == 3 {
+		// Need to have at least 3 characters to ++ or --
+		if len(parts[0]) < 3 {
+			return false
+		}
+
+		subject := strings.ToLower(nick)
+		itemName := strings.ToLower(parts[0])[:len(parts[0])-2]
```
Comment (chrissexton) [inline plugins/counter/counter.go]: Since we don't need to chop off ++ or -- here, the weird indexing math can go away.

**Comment by mccoyst** [comment]
PTAL

**Pull request merged**